### DISCUSSION TITLE
Localization possible?

### DISCUSSION THREAD

**Discussion by pataquets** [KiraLT/torrent-stream-server#219] [Ideas]
I was wondering if the used framework has any feature which allows localization to other languages and how to add one.﻿

**Comment by KiraLT** [comment]
You wish to locale only frontend I guess? Backend, in theory, has only error messages (fatal messages), so I don't think we need to translate that. I will think about the best way to implement it.

**Comment by pataquets** [comment]
Sure, I was meaning frontend. Just wanted to know in case was easy, no worries.
The issue was just a support question and can be closed, for my part.
Leaving it up to you in case you want to use it as a feature request issue.
